packages/generator/src/metadata/metadata.spec.ts

```diff
@@ -1,16 +1,16 @@
 import nock = require('nock');
 import { createOptions } from '../../test/test-util/create-generator-options';
 import { getTestService } from './pregenerated-lib.spec';
-import { sdkMetaDataJS } from './sdk-metadata';
+import { metadata } from './metadata';
```

Comment:
I think this is a fair refactoring, but I would like to see it in a seperate PR the next time so that we have smaller number of files to review.

Reply:
I agree, sorry for that. I will be more careful in the future.